server.go

```diff
@@ -236,7 +236,8 @@ func newJoinServerOption(opts ...ServerOption) ServerOption {
 // WriteBufferSize determines how much data can be batched before doing a write on the wire.
 // The corresponding memory allocation for this buffer will be twice the size to keep syscalls low.
 // The default value for this buffer is 32KB.
-// Zero will disable the write buffer such that each write will be on underlying connection.
+// Zero or negative values will disable the write buffer such that each write will be on underlying
+// connection.
```

Comment:
While you are here, would you mind line wrapping this whole docstring at 80 cols. Thank you so much.